### ISSUE TITLE
401 Authorization exception, symbols not escaping.

### ISSUE THREAD

**Issue by EMartirosyan** [linvi/tweetinvi#192]
Was working with users search recently. Everything worked perfectly, but when I passed as a search query: "C# developer". I started recieving 401 Authorization exception. And the thing is that the URL of the request contains "#" symbol, somehow it was not escaped.

**Comment by linvi** [comment]
Which version of Tweetinvi are you using? This problem should have been fixed in the latest release 0.9.11.2.

**Comment by EMartirosyan** [comment]
<package id="TweetinviAPI" version="0.9.11.2" targetFramework="net45" /> - line from packages.config
I think the thing is that "#" is actually allowed in the URL ...

**Comment by linvi** [comment]
> I think the thing is that "#" is actually allowed in the URL ...

What do you mean?

**Comment by linvi** [comment]
I do not understand what is wrong. I have tried the package and did exactly what you said it works perfectly well for me.

``` c#
// Version 0.9.11.2.

namespace ConsoleApplication2
{
    class Program
    {
        static void Main(string[] args)
        {
            Auth.SetUserCredentials("...", "...", "...", "...");
            var tweets = Search.SearchTweets("C# Developers");

            Console.ReadLine();
        }
    }
}
```

**Comment by EMartirosyan** [comment]
Tweet search works for me to, but I have issue with users search.
`Search.SearchTweets("C# software develop");
Search.SearchUsers("C# software develop");`

First line executes without issues, but when it tries to execute second I recieve 401 Authorization exception.

Here is the request URL: https://api.twitter.com/1.1/users/search.json?q=C#%20software%20develop&page=0&count=20&include_entities=True

**Comment by linvi** [comment]
Thanks I will look into it. This might be released as part of version 0.9.12 as I expect it to be released this week or next weekend.

**Comment by linvi** [comment]
Release 0.9.12.0 inconclusive as this bug fix prevent the filters to work properly,
